must-gather/Makefile

```diff
@@ -0,0 +1,8 @@
+IMAGE_REGISTRY ?=quay.io/kubevirt
+IMAGE_TAG ?= latest
+MUST_GATHER_IMAGE ?= must-gather-lifecycle
+
+all:
```

Comment:
Is there a reason for this addional Makefile? What about adding this to the root Makefile alongside the other container build targets?

Reply:
I was not quite sure were this will end up, I understood that we potentially have another operator that will be covered by the same must-gather image. Ok, meanwhile it's here, so your comment makes sense.

Reply:
additional reason: also we might have to move must-gather out some day, as there are/will be additional operators that need the same treatment.

Reply:
> I need to change the directory for building the docker image

not sure if I can follow, you can use `docker build must-gather -t ...`?

> additional reason: also we might have to move must-gather out of this repository some day

maybe, but for now I'd prefer to keep it simple. And less files are simpler imho. Makes sense?

Reply:
done